<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>engine</class>
 <widget class="QWidget" name="engine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>737</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>529</width>
    <height>737</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit"/>
       <widget class="QPushButton" name="pushButton_Enter">
        <property name="text">
         <string>Enter</string>
        </property>
       </widget>
      </widget>
      <widget class="QTextEdit" name="textEdit"/>
     </widget>
     <widget class="QPushButton" name="pushButton_Clear">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_engine">
     <property name="text">
      <string>引擎走棋</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
